src/credentials/CHIPCert.h

```diff
@@ -56,6 +56,13 @@ static constexpr uint32_t kMaxCHIPCertDecodeBufLength = kMaxDERCertLength - Cryp
 
 // Muximum number of CASE Authenticated Tags (CAT) in the CHIP certificate subject.
 static constexpr size_t kMaxSubjectCATAttributeCount = CHIP_CONFIG_CERT_MAX_RDN_ATTRIBUTES - 2;
+static constexpr uint32_t kUndefinedCAT              = 0;
```

Comment:
A typedef for uint32_t CAT type in lib/core/DataModelTypes.h seems like it would be useful.

Reply:
done: added `CASEAuthTag` type to lib/core/DataModelTypes.h